src/enums.py

```diff
@@ -124,7 +124,7 @@ class InventoryResource(_SerialisableEnum):
     TOMATO_SEED = 11
 
     def get_worth(self):
-        return self._ITEM_WORTHS[self]  # noqa
+        return self._ITEM_WORTHS[self - 2]  # noqa
```

Comment:
Why are we subtracting 2 from the items worth here?

Reply:
yeah... last week there was a problem that is when buying a seed, the game crashes and the error says out of index and I tried to subtract from the index that was out of bounds and it worked with the prices being correct, but I guess that the last commits fixed it because I recently ( 1- 3 minutes ago ) checked it and it worked without the subtraction even the subtraction made the prices wrong. So, I will omit it.